apps/family-public-law/preview/base/kustomization.yaml

```diff
@@ -7,6 +7,9 @@ resources:
   - ../../../aac/identity/identity.yaml
   - ../../../am/identity/identity.yaml
   - ../../../base/workload-identity
+  - ../../../azureserviceoperator-system/resources/flexibleserver-postgres.yaml
+  - ../../../azureserviceoperator-system/resources/flexibleserver-postgres-config.yaml
+  - ../aso/fpl-postgres-config.yaml
 #- resource-limits.yaml
```

Comment:
missing this line for  resource group 
https://github.com/hmcts/cnp-flux-config/blob/81fad921da459a6de8a658bc0e664b4a21b70cf6/apps/et/preview/base/kustomization.yaml#L14